### ISSUE TITLE
vega-viewtransforms 4.5.9 has dependency on nonexistent vega-dataflow

### ISSUE THREAD

**Issue by jeffemandel** [webjars/webjars#1986]
I have a POM that includes:
````
       <dependency>
            <groupId>org.webjars.npm</groupId>
            <artifactId>vega</artifactId>
            <version>5.22.1</version>
        </dependency>
````
This POM allows vega-dataflow 5.7.4, but it brings in vega-viewtransforms , which has
````
        <dependency>
            <groupId>org.webjars.npm</groupId>
            <artifactId>vega-dataflow</artifactId>
            <version>[5.7.5,6)</version>
        </dependency>
````
This creates a problem; the latest version of vega-dataflow in web jars is 5.7.4. Looking on GitHub I see that the version numbers have been updated; vega-dataflow 5.7.4 is now 5.7.5. I'm patching my local repository, but it would be nice if this could be addressed.

**Comment by jamesward** [comment]
Thanks for the report!  Were you able to deploy the missing dep from https://webjars.org or did that fail for some reason?

**Comment by jeffemandel** [comment]
I edited the vega-viewtransforms POM in my local repository to reflect the current versions of the two dependencies. This worked, but the better solution would be to bump the versions for vega-dataflow and vega-scenegraph to the proper versions in GitHub.

Thanks

**Comment by jamesward** [comment]
Makes sense.  You should be able to deploy these yourself from https://webjars.org  Can you try that and report back if there are issues?

**Comment by jeffemandel** [comment]
As best I understand from the vega GitHub history, the only change in the jars are the version number, but I didn’t unpack the webjar to confirm this. All of my Vega visualizations work, so I didn’t pursue it further. I suspect that all you need to do is rebuild the vega web jars from the current version and everything will be fine.

**Comment by jamesward** [comment]
You can rebuild and deploy the webjars on your own via https://webjars.org Just click the + button. I think that should fix the issue.
